cluster/mesos/docker/mesos-slave/bin/wrapdocker

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+# Copyright 2013-2014 Jérôme Petazzoni, Johan Haleby, lalyos, James Harris,
+#                     Michael Elsdörfer, Tony Hesjevik, Esben Haabendal,
+#                     Michael A. Smith
+# Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
But in actuality, `cluster/ubuntu` is the only directory so far we've let through with scripts that weren't .sh, so.. I guess I should ask you to change everything to be consistent. I'm sorry. I'm so sorry.